src/escalus_event.erl

```diff
@@ -91,15 +91,44 @@ write_events([], _) ->
 write_events(Events, OutFileName) ->
     {ok, FD} = file:open(OutFileName, [write]),
     BaseTime = base_time(Events),
+    %% What is ../../../.. ?
```

Comment:
it's kinda CT config-dependent.
But we and our clients usually use these 4 levels of directories.